dataprofiler/__init__.py

```diff
@@ -25,8 +27,22 @@
         ImportWarning
     )
 
+# Initialize logging config
+logging.basicConfig(level=logging.INFO)
+
+
 def set_seed(seed=None):
     # also check it's an integer
     if seed is not None and (not isinstance(seed, int) or seed < 0):
         raise ValueError("Seed should be a non-negative integer.")
     settings._seed = seed
+
+
+def set_verbosity(verbose=True):
+    if verbose:
+        # INFO level will print/log update strings and other outputs as well
+        # as warnings/errors
+        logging.getLogger().setLevel(logging.INFO)
+    else:
+        # WARNING level will print/log only warnings/errors
+        logging.getLogger().setLevel(logging.WARNING)
```

Comment:
I think we should take in / allow verbosity levels. 
e.g. in TensorFlow they allow you to get the actual logger and set the level yourself:
e.g. `dp.get_logger().set_level('INFO')`

This also allows someone to change its handlers or even where it's outputting.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/platform/tf_logging.py
Also, we may want to consider putting this in its own file and importing it in `__init__.py`

Reply:
So do you want to maintain the simple `True`/`False` interface for quick enabling/disabling or just have them take the level, because then this essentially just becomes a wrapper for `logging.getLogger.setLevel` that doesn't add anything else.

Reply:
getLogger abstracts the variable from the user and allows them to alter the logger for more functionality as illustrated in the issue.

Also, as seen in the tag case, allows them to make it thread safe I believe.

Reply:
Also, with new file, logger could be saved as a variable abstracted from the user.

Reply:
Not sure this is best yet.